ECOSYSTEM.md

```diff
@@ -38,9 +47,15 @@ Core contributors are individuals who contribute to the React Native project. A
 
 These are behaviors we have observed in our existing core contributors. They aren't strict rules but rather outline their usual responsibilities. We do not expect every core contributor to do all of the above things all the time. Most importantly, we want to create a supportive and friendly environment that fosters collaboration. Above all else, **we are always polite and friendly.**
 
-Core contributor status is attained after consistently contributing and taking on the responsibilities outlined above and granted by other core contributors. Similarly, after a long period of inactivity, a core contributor may be removed.
+Core contributor status is attained after consistently contributing and taking on the responsibilities outlined above and granted by other core contributors. Similarly, after a long period of inactivity (~6 months or more), a core contributor may be contacted to understand if they’re still interested in being part of the program.
+
+You can use this form <MISSING_LINK_TODO> to either:
+* Apply yourself to become a Core Contributor. Make sure to include a list of valuable contributions you did to the React Native repository and ecosystem.
+* Nominate someone to become a Core Contributor.
+
+As a core contributor, you will have access to the core contributor Discord which is used for light-weight coordination and discussion. 
 
-We aim to make contributing to React Native as easy and transparent as possible. All important topics are handled through a [discussion or RFC process on GitHub](https://github.com/react-native-community/discussions-and-proposals). We are always looking for active, enthusiastic members of the React Native community to become core contributors.
+**We aim to make contributing to React Native as easy and transparent as possible.** All important topics are handled through a [discussion or RFC process on GitHub](https://github.com/react-native-community/discussions-and-proposals). We are always looking for active, enthusiastic members of the React Native community to become core contributors.
```

Comment:
not sure if here in the examples of where we aim for transparent and open communication you might want to add the releases WG (https://github.com/reactwg/react-native-releases) and the new arch WG (https://github.com/reactwg/react-native-new-architecture)